holoclean.py

```diff
@@ -164,14 +164,35 @@ def __init__(self, env, name="session"):
         self.repair_engine = RepairEngine(env, self.ds)
         self.eval_engine = EvalEngine(env, self.ds)
 
-    def load_data(self, name, f_path, f_name, na_values=None):
-        status, load_time = self.ds.load_data(name, f_path,f_name, na_values=na_values)
+    def load_data(self, name, fpath, na_values=None,
+            entity_col=None, src_col=None):
+        """
+        load_data takes the filepath to a CSV file to load as the initial dataset.
+
+        :param name: (str) name to initialize dataset with.
+        :param fpath: (str) filepath to CSV file.
+        :param na_values: (str) value that identifies a NULL value
+        :param entity_col: (st) column containing the unique
+            identifier/ID of an entity.  For fusion tasks, rows with
+            the same ID will be fused together in the output.
+            If None, assumes every row is a unique entity.
+        :param src_col: (str) if not None, for fusion tasks
+            specifies the column containing the source for each "mention" of an
+            entity.
+        """
+        status, load_time = self.ds.load_data(name, fpath, na_values=na_values,
+                entity_col=entity_col, src_col=src_col)
         print(status)
         if self.env['verbose']:
             print('Time to load dataset: %.2f secs'%load_time)
 
-    def load_dcs(self, f_path, f_name):
-        status, load_time = self.dc_parser.load_denial_constraints(f_path, f_name)
+    def load_dcs(self, fpath):
```

Comment:
I think we'll need to eventually need to enforce versioning for these backwards-incompatible public API changes. We should have a separate release/stable branch, whereas master can be the most up-to-date non-stable version. Presumably people who will be developing/experiment with the master branch will understand that the public interfaces are free to change at any time. A release branch will also let us build pip/conda packages using git references.

I believe this is how most open source projects structure releases.

Let me know if you still want this to be reverted. I think it should be fine for now since we are merging into the `dev` branch. It also lets us fix this ugliness of separate `f_fpath/f_name` parameters early on instead of pushing it off indefinitely and not get around to it.

Reply:
Okay cool